loader_hub/feedly_rss/base.py

```diff
@@ -0,0 +1,56 @@
+"""Feedly Rss Reader"""
+
+from pathlib import Path
+from feedly.api_client.session import FeedlySession
```

Comment:
oh since it's failing tests, the one comment i was going to make is to move this to be a lazy import (e.g. in `load_data`)